docker/docker-compose.yaml

```diff
@@ -73,3 +77,4 @@ services:
 
 volumes:
     postgres-data: {}
+    tmp: {}
```

Comment:
Can you pick something more appropriate and easier to grep for? Maybe something with  "persistence" or "export" or the like?